frontend/src/pages/modelRegistry/screens/ModelPropertiesTableRow.tsx

```diff
@@ -144,7 +146,21 @@ const ModelPropertiesTableRow: React.FC<ModelPropertiesTableRowProps> = ({
             onToggle={(_event, isExpanded) => setIsValueExpanded(isExpanded)}
             isExpanded={isValueExpanded}
           >
-            {value}
+            {isValidHttpUrl(value) ? (
```

Comment:
It could be nice to encapsulate this functionality in its own component for future reuse, just a small suggestion, don't worry about it right now :)